### ISSUE TITLE
feat: Custom handle properties

### ISSUE THREAD

**Issue by hadi7khan** [cgutierr-zgz/image_compare_slider#11]
We must be able to change the handle color. For example divider is of blue color and handle is of black color.

We must be change the divider position to negative as well like take divider and handle out of ImageCompareSlider, from top or bottom. Just like image added below.
![Screenshot from 2023-08-02 17-23-37](https://github.com/cgutierr-zgz/image_compare_slider/assets/68601505/54e9be58-be4d-4eda-b7bd-e98e19f97636)

**Comment by cgutierr-zgz** [comment]
Hi, @hadi7khan 
I've just updated the library so you can customize the handle color and the handle outline, check the 2.8.0
 version at: https://pub.dev/packages/image_compare_slider/versions/2.8.0

<img width="298" alt="Screenshot 2023-08-02 at 17 32 42" src="https://github.com/cgutierr-zgz/image_compare_slider/assets/18635739/aa84e854-d47c-42d1-ac85-8e4566a0b221">

**Comment by hadi7khan** [comment]
Thank you for quick response. That's really great. I appreciate your efforts. What about letting divider and handle move out of ImageCompareSlider? like in the image above. Is it possible to achieve? maybe with negative handle position and divider must follow hande?

**Comment by cgutierr-zgz** [comment]
> Thank you for quick response. That's really great. I appreciate your efforts. What about letting divider and handle move out of ImageCompareSlider? like in the image above. Is it possible to achieve? maybe with negative handle position and divider must follow hande?

I'm pretty sure there should be a way right now by using the item builders, but I'd need to investigate into that, right now im on vacation and might not have that much time

Also I'd investigate to add shadows to the divider/handle for the future

**Comment by hadi7khan** [comment]
That's great. Enjoy your vacation, let me know when you work on it. While you're at it, you can let users add image on handle as well. Maybe someone wants to add their dp's on it.

**Comment by cgutierr-zgz** [comment]
> That's great. Enjoy your vacation, let me know when you work on it. While you're at it, you can let users add image on handle as well. Maybe someone wants to add their dp's on it.

Yeah, theres already a ticket (#2) but that will prolly take me quite some time, im maybe thinking on rewriting it entirely because there's many things i'd like to change hahaha

Anyway, I'm going to add that suggestion :)

**Comment by hadi7khan** [comment]
Alright, Also add aspectRatio to it. Users might want to add aspectRatio to this widget or single image. If i wrap ImageCompareSlider with AspectRatio it is breaking.

**Comment by hadi7khan** [comment]
Any update on this one?

**Comment by cgutierr-zgz** [comment]
> Any update on this one?

Pretty busy lately with many other projects, feel free to have a look if you have some time, if not i'll postpone this until i have more availability :(
